Analysis/HiggsTauTauRun2/src/HTTPairGenInfo.cc

```diff
@@ -75,6 +75,8 @@ TVector3 GenIP (ic::GenParticle *h, ic::GenParticle *t) {
       if(era_ == era::data_2016UL_postVFP) graph_name+="_2016UL_postVFP";
       if(era_ == era::data_2017UL) graph_name+="_2017UL";
       if(era_ == era::data_2018UL) graph_name+="_2018UL";
+      if(era_ == era::data_2022_preEE) graph_name+="_2022_preEE";
```

Comment:
I dont think we measure this correction for 2022 so this graph should either not get used or pick up the graph used for 2018